config/policy.go

```diff
@@ -20,9 +21,9 @@ type Policy struct {
 }
 
 // NewPolicy return policy config with specified backend preset
-func NewPolicy(backend string) *Policy {
+func NewPolicy() *Policy {
 	return &Policy{
-		Backend: backend,
+		Backend: ceph.BackendName,
```

Comment:
My previous comment might not be very clear. I am wondering if we shall just accept backend at time of `volume create` and default to `ceph` just there?